### ISSUE TITLE
Assess and remove duplicate "Source system" taxonomy

### ISSUE THREAD

**Issue by NicoledeGreef** [bcgov/MFIN-Data-Catalogue#458]
**OP timer**

https://openplus.monday.com/boards/4092908516/pulses/6552899330

---

It was determined that the best thing to do is remove `source_system` vocabulary.

- [x] Remove `source_system` vocabulary | see [this comment](#issuecomment-2088717340)
## Tasks

~- [x] ensure the field `source system` is using the correct **Source system** taxonomy~
~- [x] resolves any errors that result from switching the taxonomy associated with **Source system**~
~- [x] check any Views where the old taxonomy is being used and swap/remove it applicable~
~- [x] check search facets to ensure the correct taxonomy is being used~
~- [x] update `source_system` taxonomy with new values that have been added to `data_system`~

~## Deployment steps~

~1. - [ ] export all Test and Prod records using the report builder, this is to get the current `source_system` values, note the values~
~2. - [ ] delete existing `source_system` values on prod~
~3. - [ ] delete vocabulary `data_system` on dv14, dev, test and prod. **This has been relabelled to _DELETE | source system_**~
~4. - [ ] deploy~
~5. - [ ] once deployment is complete, manually update the records to match the values @NicoledeGreef~

**Comment by CraigClark** [comment]
In tests, we need to find records that use the source system, delete the value, then we can add it back after config is done

In ddev I tried using the content overview vbo to set source_system to none, it didn't work. Noting here, I'll try in dv14

**Comment by CraigClark** [comment]
New issue created from this task #459

**Comment by CraigClark** [comment]
~~For search facts, I'm seeing a facet called `blob`. I believe this is happening because there are values in the `source_system` field that are not in the taxonomy. This problem shouldn't occur if we empty out the values currently used by _Source system_ .~~

Resolved. This was an issue with an orphaned term. IT has been resolved on dv14 and prod

**Comment by CraigClark** [comment]
@lkmorlan 

Ready for review/merge https://github.com/bcgov/MFIN-Data-Catalogue/tree/458-remove-duplicate-source-system-taxonomy

before we can import the config, we need to delete the vocabulary `data_system`

**Comment by CraigClark** [comment]
@NicoledeGreef before we commit this, I would like to confirm that `source_system` is the vocabulary you want to keep and not `data_system`.

@lkmorlan  recalls a discussion where we were going to go with `data_system` because that's what the term represents. Then we could reuse it elsewhere. So think if it like "The source system for this data is this data system", where `field_source_system` on the metadata record gets populated by values from the `data_system` vocabulary.  Then in the future, `data_system` is available for other things and it's easy to understand because `data_system` is just a list, sources or not. 

Please let me know what taxonomy you want to keep.

If you want to use `source_system` we will deploy this change. The work is done.

If you want to keep `data_sysyem`, it's even easier since `field_source_system` already uses it, all we need to do is delete `source_system` and re-label `data_system` for clarity.

**Comment by NicoledeGreef** [comment]
if the underlying name `data_system` is the one to keep let's do that and keep the **Source system** and Source system docs.

**Comment by CraigClark** [comment]
@lkmorlan ready for review/merge https://github.com/bcgov/MFIN-Data-Catalogue/tree/458-remove-source-system-vocab-and-update-data-system

**Comment by lkmorlan** [comment]
Merged. The terms in `source_system` need to be removed before deployment.
